### PR TITLE
fix(cli): deduplicate classNames in applyColorMapping

### LINKED ISSUE #692: Select and TextArea components duplicated border classes
Hi, I love the new update, it's awesome ❤️ 

I found that the `border-neutral-200` and `dark:border-neutral-800` classes on the `select.tsx` component are duplicated.

I'm using `"cssVariables": false`.

image

image

### LINKED ISSUE #905: Duplicate tailwind classnames after using add command
I keep getting duplicate tailwind classnames after running add <component here>
For example

`npx shadcn-ui@latest add checkbox`

Then in my checkbox.tsx, there's duplicate border color `border-gray-200` and `border-gray-900`.

image

There's also the same duplicate border color for the dark variant `dark:border-gray-800` and `dark:border-gray-50`

image

This also happens with other components as well.

### PR DESCRIPTION
**Pull request by santidalmasso**
Deduplicate class names within the `applyColorMapping` function. To resolve this, the method of storing class names has been updated from using arrays to sets, which inherently ensures that all class names are unique.

- Closes #692
- Closes #905

### CODE REVIEW

File: packages/cli/src/utils/transformers/transform-css-vars.ts [hunk 1]
```diff
@@ -146,38 +146,42 @@ export function applyColorMapping(
...
-    if (!lightMode.includes(className)) {
-      lightMode.push(className)
+    if (!lightMode.has(className)) {
+      lightMode.add(className)
     }
   }
 
-  return lightMode.join(" ") + " " + darkMode.join(" ").trim()
+  return (
+    Array.from(lightMode).join(" ") +
```
Comment (yanisneverlies) [inline packages/cli/src/utils/transformers/transform-css-vars.ts:183]: 🔧 Can be simplified to `return [...lightMode, ...darkMode].join(' ').trim()`.

**Comment by shadcn** [comment]
@santidalmasso Thanks for the PR. Reviewing...

File: packages/cli/src/utils/transformers/transform-css-vars.ts (hunk 1 above)
Comment (yanisneverlies) [reply packages/cli/src/utils/transformers/transform-css-vars.ts]: I like the solution you shipped with the latest commit though. Much cleaner now!

**Review by shadcn — APPROVED** [review]
THANK YOU.

**Pull request merged**